### PR TITLE
Add more python questions

### PR DESCRIPTION
**Pull request by Mannuel25**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

### CODE REVIEW

File: src/data/python-quiz.ts [hunk 1]
```diff
@@ -1143,6 +1143,69 @@ const pythonQuiz = [
...
+    Question: "What is the output of print(math.pow(3,4))?",
+    Answer: "81.0", 
+    Distractor1: "81", 
+    Distractor2: "81.0", 
+    Distractor3: "34", 
+    Explanation: "By the default math.pow() converts both its arguments to type float", 
+    Link: "https://docs.python.org/3/library/math.html#power-and-logarithmic-functions", 
+  },
+  { 
+    Question: "Which of the following is not a tuple method?",
```
Comment (jdwilkin4) [inline src/data/python-quiz.ts:1165]:
```suggestion
    Question: "In Python, which of the following is not a tuple method?",
```

File: src/data/python-quiz.ts [hunk 2]
```diff
@@ -1143,6 +1143,69 @@ const pythonQuiz = [
...
+    Answer: "insert", 
+    Distractor1: "count", 
+    Distractor2: "insert", 
+    Distractor3: "index", 
+    Explanation: "There are only two tuple methods; count and index", 
+    Link: "https://docs.python.org/3/tutorial/datastructures.html", 
+  },
+  { 
+    Question: "What will be the data type of 'John Doe'.split()?",
```
Comment (jdwilkin4) [inline src/data/python-quiz.ts:1174]:
```suggestion
    Question: "In Python, what will be the data type of 'John Doe'.split()?",
```

File: src/data/python-quiz.ts [hunk 3]
```diff
@@ -1143,6 +1143,69 @@ const pythonQuiz = [
...
+    Answer: "List", 
+    Distractor1: "string", 
+    Distractor2: "List", 
+    Distractor3: "tuple", 
+    Explanation: "The result will be ['John', 'Doe'], which has a list data type", 
+    Link: "https://www.freecodecamp.org/news/how-to-split-a-string-in-python/", 
+  },
+  { 
+    Question: "If my_nums = [8,9,10], which of the following can't be used to reverse the list?",
```
Comment (jdwilkin4) [inline src/data/python-quiz.ts:1183]:
```suggestion
    Question: "In Python, if my_nums = [8,9,10], which of the following can't be used to reverse the list?",
```

File: src/data/python-quiz.ts [hunk 4]
```diff
@@ -1143,6 +1143,69 @@ const pythonQuiz = [
...
+    Answer: "reverse(my_nums)", 
+    Distractor1: "my_nums.reverse()", 
+    Distractor2: "reverse(my_nums)", 
+    Distractor3: "my_nums[::-1]", 
+    Explanation: "reverse isn't a defined Python keyword", 
+    Link: "https://www.freecodecamp.org/news/python-reverse-list-reversing-an-array-in-python/#reversed-function", 
+  },
+  { 
+    Question: "Which built-in function returns all properties and methods of the specified object, without the values?",
```
Comment (jdwilkin4) [inline src/data/python-quiz.ts:1192]:
```suggestion
    Question: "In Python, which built-in function returns all properties and methods of the specified object, without the values?",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by Mannuel25** [comment]
Thanks for the corrections, I'll do well to fix them all.

**Comment by Mannuel25** [comment]
Hi @jdwilkin4 , I've made the necessary corrections.

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

Also, in the future, it is best to create a branch for new changes.
Please read through the contributing docs on how to do that.

**Pull request merged**